docs/tutorials/getting_started/command_line.rst

```diff
@@ -0,0 +1,45 @@
+Using the command line tool
+===========================
+
+Once :code:`axelrod` is installed you have access to a `run_axelrod` script that
+can help run some of the tournaments, include the tournament that involves all
+of the strategies from the library. You can view them on this repository:
+`https://github.com/Axelrod-Python/tournament/`_
+
+To view the help for the :code:`run_axelrod` file run::
+
+    $ run_axelrod.py -h
+
+Note that if you have not installed the package you can still used this script
+directly from the repository::
+
+    $ python run_axelrod -h
+
+There are a variety of options that include:
```

Comment:
Is that correct? 'There are a variety of options' reads better to me... 

Just googled and apparently it's non a done thing: http://thegrammarexchange.infopop.cc/eve/forums/a/tpc/f/340600179/m/9861036123

I don't care too strongly, just checking.